### PR TITLE
nextflow mvp

### PR DESCRIPTION
**Pull request by kennyworkman**
rage
rage
against the dying of the light

### CODE REVIEW

File: latch_cli/nextflow/workflow.py [hunk 1]
```diff
@@ -0,0 +1,1181 @@
...
+    ):
+        global_start_node = Node(
+            id=_common_constants.GLOBAL_INPUT_NODE_ID,
+            metadata=None,
+            bindings=[],
+            upstream_nodes=[],
+            flyte_entity=None,
+        )
+
+        def create_bindings(
```
Comment (ayushkamat) [inline latch_cli/nextflow/workflow.py:153]: move this and `global_start_node` to top level

File: latch_cli/nextflow/workflow.py [hunk 2]
```diff
@@ -0,0 +1,1181 @@
...
+                    bindings=bindings,
+                    upstream_nodes=upstream_nodes,
+                    flyte_entity=operator_task,
+                )
+
+                node_map[vertex.id] = node
+
+            elif vertex.vertex_type == VertexType.origin:
+                # generic channel
+                ...
```
Comment (ayushkamat) [inline latch_cli/nextflow/workflow.py:361]: del, make it a todo if you plan on supporting this in the future

**Review by ayushkamat — CHANGES_REQUESTED** [review]
didnt look too closely at the shared logic

**Review by rahuldesai1 — COMMENTED** [review]
minor comments

have some questions about how the branching works, mostly for my understanding
also don't really understand NextflowProcessPost/PreAdapterTask

lets sync on Monday

**Pull request closed without merging**